app/components/candidate_interface/contact_details_review_component.rb

```diff
@@ -2,21 +2,26 @@ module CandidateInterface
   class ContactDetailsReviewComponent < ViewComponent::Base
     validates :application_form, presence: true
 
-    def initialize(application_form:, editable: true, missing_error: false)
+    def initialize(application_form:, editable: true, missing_error: false, submit_application_attempt: false)
```

Comment:
Would `submitting_application` be a better name for the new option? I'm not sure.

Reply:
yea i actually prefer submitting_application 

Thanks for the suggestion.